python/src/aac/plugins/gen_plant_uml/gen_plant_uml_impl.py

```diff
@@ -19,9 +20,10 @@
 COMPONENT_STRING = "component"
 OBJECT_STRING = "object"
 SEQUENCE_STRING = "sequence"
+INVALID_FILE_NAME_CHARACTERS = ".!@#$%^&*();,\\/?[]{}`~|'"
```

Comment:
I think this can be clearer, especially since periods are valid file name characters.
```suggestion
FILE_NAME_CHARACTERS_TO_REPLACE = ".!@#$%^&*();,\\/?[]{}`~|'"
```